site/docs/master/configuration.md

```diff
@@ -1,9 +1,54 @@
 # Contour Configuration
 
+## Configuration File
+
 A configuration file can be passed to the `contour serve` command which specified additional properties that Contour should use when starting up.
 This file is passed to Contour via a ConfigMap which is mounted as a volume to the Contour pod.
 
-Following is an example ConfigMap with configuration file included: 
+The Contour configuration file is optional.
+In its absence, Contour will operate with reasonable defaults.
+
+| Field Name | Type | Description |
+|------------|------|-------------|
```

Comment:
Should we add a "defaults" column to explain what the defaults are for these values?

Reply:
I'll add this to see what you think

Reply:
Adding defaults looks OK to me.